pkg/agent/openflow/framework.go

```diff
@@ -152,6 +152,10 @@ func newTable(tableName string, stage binding.StageID, pipeline binding.Pipeline
 	return table
 }
 
+func (t *Table) GetStatus() bool {
```

Comment:
perhaps name it `IsInitialized`

Reply:
Done, probably could wait until 5369 is merged.